qiskit/circuit/quantumcircuit.py

```diff
@@ -3386,6 +3386,91 @@ def depth(
 
         return max(op_stack)
 
+    def depth_2q(self, include_directives: bool = False) -> int:
+        """Returns the depth in terms of quantum gates with at least 2-qubits,
+        excluding directives by default.
+
+        Args:
+            include_directives (bool): Whether to include directives such as barriers
+
+        Returns:
+            int: Quantum circuit depth in terms of two-qubit gates
+        """
+        if not include_directives:
+            return self.depth(
+                lambda x: not getattr(x.operation, "_directive", False) and x.qubits >= 2
+            )
+        else:
+            return self.depth(lambda x: x.qubits >= 2)
+
+    def depth_nq(self, n: int, include_directives: bool = False) -> int:
+        """Returns the depth in terms of quantum gates with at least n-qubits,
+        excluding directives by default.
+
+        Args:
+            n (int): The minimum number of qubits in an instruction that counts towards the depth.
+            include_directives (bool): Whether to include directives such as barriers
+        Returns:
+            int: Quantum circuit depth in terms of n-qubit gates.
+        """
+        if not include_directives:
+            return self.depth(
+                lambda x: not getattr(x.operation, "_directive", False) and x.qubits >= n
+            )
+        else:
+            return self.depth(lambda x: x.qubits >= n)
+
+    def num_2q_gates(self, include_directives: bool = False) -> int:
+        """Returns the number of gates with exactly two qubits, excluding directives by default.
+
+        Args:
+            include_directives (bool): Whether to include directives such as barriers
+
+        Returns:
+            int: The number of gates in the quantum circuit with exactly two qubits.
+        """
+        if not include_directives:
+            return self.size(
+                lambda x: not getattr(x.operation, "_directive", False) and x.qubits == 2
+            )
+        else:
+            return self.size(lambda x: x.qubits == 2)
+
+    def num_nq_gates(self, n: int, include_directives: bool = False) -> int:
```

Comment:
Same suggestion: have `n:int = 2` and drop the `num_2q_gates`

Reply:
Thanks for your suggestion. I removed the `n=2` versions of the methods and updated the release notes accordingly!